apps/web/package.json

```diff
@@ -9,7 +9,8 @@
     "analyze": "ANALYZE=true next build",
     "dev": "node --require ./tracer/initialize.js ./node_modules/.bin/next dev",
     "lint": "next lint",
-    "update-contributors": "node ./scripts/updateContributors.js && npx prettier ./src/components/CoreContributors/CoreContributors.json -w"
+    "update-contributors": "node ./scripts/updateContributors.js && npx prettier ./src/components/CoreContributors/CoreContributors.json -w",
+    "fetch-mirror-blog": "node ./scripts/pullLatestBlogPosts.js"
```

Comment:
what was our prior pattern? manual? why is this necessary for the new version?

Reply:
no prio pattern to load blog post, this is a helpful tasks to keep homepage up to date.